x11-themes/kvantum/kvantum-1.1.2.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+QT5MIN="5.15.0"
+QT6MIN="6.2.0"
+KF5MIN="5.82.0"
+
+inherit cmake xdg
+
+DESCRIPTION="SVG-based theme engine for Qt5, KDE Plasma and LXQt"
+HOMEPAGE="https://github.com/tsujan/Kvantum"
+SRC_URI="https://github.com/tsujan/${PN^}/archive/V${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN^}-${PV}/${PN^}"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc64 ~x86"
+IUSE="qt6"
+
+RESTRICT="test" # no tests
+
+CDEPEND="
+	>=dev-qt/qtcore-${QT5MIN}:5
+	>=dev-qt/qtgui-${QT5MIN}:5
+	>=dev-qt/qtsvg-${QT5MIN}:5
+	>=dev-qt/qtwidgets-${QT5MIN}:5
+	>=dev-qt/qtx11extras-${QT5MIN}:5
+	>=kde-frameworks/kwindowsystem-${KF5MIN}:5
+	x11-libs/libX11
+	qt6? (
+		>=dev-qt/qtbase-${QT6MIN}:6[gui,widgets]
+		>=dev-qt/qtsvg-${QT6MIN}:6
+	)
+"
+DEPEND="
+	${CDEPEND}
+	x11-base/xorg-proto
+"
+RDEPEND="
+	${CDEPEND}
+"
```

Comment:
`CDEPEND` seems unnecessary as `CDEPEND` and `RDEPEND` are identical. You can just inherit the contents of `RDEPEND` into `DEPEND`. 

And try not to use a single-char`DEPEND` but something like `COMMON_DEPEND` instead. (I know you copy-pasted an older ebuild, but in future)
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Avoid_using_single-char_.2ADEPEND_variable